subsys/debug/coredump/Kconfig

```diff
@@ -144,4 +144,27 @@ module = DEBUG_COREDUMP
 module-str = coredump
 source "subsys/logging/Kconfig.template.log_config"
 
+config DEBUG_COREDUMP_THREAD_STACK_TOP
+	bool "Dump top of stack only"
+	default y if DEBUG_COREDUMP_MEMORY_DUMP_MIN
```

Comment:
I'd like to see default y for DEBUG_COREDUMP_MEMORY_DUMP_THREADS as well

Reply:
Ok, I would prefer to wait for other comments with regards to that. For `DEBUG_COREDUMP_MEMORY_DUMP_MIN` it seems reasonable to enable it by default because a user of `DEBUG_COREDUMP_MEMORY_DUMP_MIN` surely cares about the core dump size, but I'm so sure about users of `DEBUG_COREDUMP_MEMORY_DUMP_THREADS` and it's usually better to avoid potentially unneeded default behavior changes (from my Zephyr update experience ;)).

Reply:
All right, that's reasonable